tests/worksheets/src/test/scala/tests/worksheets/WorksheetSuite.scala

```diff
@@ -226,7 +257,13 @@ class WorksheetSuite extends BaseSuite {
     """|
        |<val x = "foobar".stripSuffix("bar")> // : String = "foo"
        |x: String = "foo"
-       |""".stripMargin
+       |""".stripMargin,
+    compat = Map(
+      "0." ->
+        """|<val x = "foobar".stripSuffix("bar")> // : String = foo
+           |x: String = foo
+           |""".stripMargin
+    )
   )
 
   // test doesn't actually work currently
```

Comment:
ha it looks like this test actually works but was not updated to include types in expected output, can we make it work? :D

Reply:
Must have started to work after I removed sourcecode dependency. Updated it to run.